### ISSUE TITLE
Random Daily Reminders

### ISSUE THREAD

**Issue by Tursu311** [Demizo/Daily_You#50]
Ability to put the daily reminder not on a fixed hour but rather on a random hour or timeframe during the day, as in proprietary app BeReal

**Comment by Demizo** [comment]
Good idea! I definitely want to add this

**Comment by trixter13** [comment]
Would love to have this - I got this app as a BeReal alternative but am missing the random time aspect. I end up just never doing my log because it's just a reminder that it's noon and doesn't feel special.

**Comment by Demizo** [comment]
> Would love to have this - I got this app as a BeReal alternative but am missing the random time aspect. I end up just never doing my log because it's just a reminder that it's noon and doesn't feel special.

Agreed, I don't think I have made an entry due to the notification ever. Last time I attempted this I hit some technical challenges but that was over a year ago so it is probably worth another attempt. Also, in case you haven't stumbled across it, you can adjust the notification time (obviously doesn't solve the issue, but an end of day reminder may be more meaningful in the meantime).
